package.json

```diff
@@ -1,11 +1,14 @@
 {
   "name": "react-html-element",
   "version": "0.0.1",
-  "description": "An extension of HTMLElement that allows for easy creation of Web Components using React.",
-  "main": "index.js",
+  "description": "An HTMLElement extension that provides a simple setup for Web Components using React.",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "scripts": {
+    "test": "npm run build",
+    "build": "npm run lint && tsc",
```

Comment:
Should `lint` be part of the build step? I would think at this point everything was fine. If `lint` should be bundled with a step in the travis build process I would think it would be on `test`.

Reply:
That makes sense! I'll flip those.